tests/Config/RepositoryTest.php

```diff
@@ -155,9 +155,20 @@ public function testSetArray()
         $this->repository->set([
             'key1' => 'value1',
             'key2' => 'value2',
+            'key3',
```

Comment:
`key3` here is not really a key but a value, this is the same as doing `[0 => 'key3']`